components/application-broker/internal/broker/provision.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/pkg/errors"
 	osb "github.com/pmorie/go-open-service-broker-client/v2"
 	"github.com/sirupsen/logrus"
+	istioauthenticationalpha1 "istio.io/api/authentication/v1alpha1"
+	istiov1alpha1 "istio.io/client-go/pkg/apis/authentication/v1alpha1"
+	istioversionedclient "istio.io/client-go/pkg/clientset/versioned"
```

Comment:
(nit): IMO it makes sense to separate the imports from k8s.io and club it with other third-party imports.